### DISCUSSION TITLE
Could odin support shadow stacks by default

### DISCUSSION THREAD

**Discussion by xzores** [odin-lang/Odin#3599] [Ideas/Requests]
Could odin support shadow stacks by default? and when I say support I mean turned on.

This would make odin a very much goto for security, then if you are not connecting the app to the internet you could turn it off for preformence reasons, but I think it would be a step in the right direction.

**Comment by diimdeep** [comment]
cool idea

**Comment by xzores** [comment]
Compared to what other are trying to do to keep their programs safe a shadow stack seems like a solid idea.

https://www.youtube.com/watch?v=FSVnN6-xr9k&t
